threadit-api/Controllers/v1/UserSettingsController.cs

```diff
@@ -0,0 +1,34 @@
+using Microsoft.AspNetCore.Mvc;
+using ThreaditAPI.Models;
+using ThreaditAPI.Services;
+
+namespace ThreaditAPI.Controllers.v1
+{
+    [ApiController]
+    [Route("v1/userSettings")]
+    public class UserSettingsController : ControllerBase
+    {
+
+        [HttpGet("remove/{userId}/{spoolName}")]
+        public async Task<IActionResult> RemoveSpoolUser([FromRoute] string userId, [FromRoute] string spoolName, [FromServices] UserSettingsService userSettingsService)
```

Comment:
- add `[AuthenticationRequired]` attribute
- remove {userId} from path and just get the userId from `request.GetUser()`